### ISSUE TITLE
Capítulos faltando no livro 1 João na versão SQL na versão ACF

### ISSUE THREAD

**Issue by carmolim** [thiagobodruk/biblia#20]
Estou utilizando a versão ACF em um banco de dados e fiz uma consulta que teve um retorno vazio: 1 João 4:8. Quando fui verificar o livro de 1 João e ele só vai o capítulo 3 versículo 24 e deveria ir até 5:21.

**Comment by Moraes-cmd** [comment]
Exatamente! Queria muito que isso fosse corrigido ou adicionado, caso me dessem acesso ao código eu poderia ao menos tentar adicionar oque falta
